conference.js

```diff
@@ -1548,7 +1548,6 @@ export default {
                 }
 
                 APP.store.dispatch(localParticipantRoleChanged(role));
-                APP.API.notifyUserRoleChanged(id, role);
```

Comment:
Is this notification sent some other way?

Reply:
Via the line above:                 APP.store.dispatch(localParticipantRoleChanged(role));
which fires PARTICIPANT_ROLE_CHANGED, https://github.com/jitsi/jitsi-meet/blob/01a731ab4d80ba8a4a5232ed127942cb2c66384a/react/features/external-api/middleware.ts#L236

Reply:
Eggscellent!

Reply:
That is what it fixes as we currently fire two times the iframeAPI event about role change.